### ISSUE TITLE
4.3.3 is a tragic mess that makes no sense and needs to be cleaned

### ISSUE THREAD

**Issue by jmanico** [OWASP/ASVS#1045]
See da title

**Comment by cmlh** [comment]
[ASVS v4 4.3.3](https://github.com/OWASP/ASVS/blob/master/4.0/en/0x12-V4-Access-Control.md#v43-other-access-control-considerations) states "_Verify the application has additional authorization (such as step up or adaptive authentication) for lower value systems, and / or segregation of duties for high value applications to enforce anti-fraud controls as per the risk of application and past fraud._"

[ASVS v4 4.3.3](https://github.com/OWASP/ASVS/blob/master/4.0/en/0x12-V4-Access-Control.md#v43-other-access-control-considerations) refers to testing antifraud controls such as those provided by [Clearbit Risk](https://clearbit.com/risk) et al.

According to [git blame](https://github.com/OWASP/ASVS/blame/daa1f7291125ac95017648df51fcd7653753db4d/4.0/en/0x12-V4-Access-Control.md#L25) @danielcuthbert committed the initial text.

**Comment by jmanico** [comment]
Current text: | **4.3.3** | Verify the application has additional authorization (such as step up or adaptive authentication) for lower value systems, and / or segregation of duties for high value applications to enforce anti-fraud controls as per the risk of application and past fraud. | | ✓ | ✓ | 732 |

**Comment by cmlh** [comment]
Can @OWASP let me know the reason that my comment was deleted as per https://github.com/OWASP/ASVS/issues/1045#event-5233270770 please?

**Comment by tghosth** [comment]
Hi @jmanico,

Having read through this requirement, I agree it needs changing. It looked like you had planned to delete it but then you did not, do you remember why?

Otherwise, I think we need reword this to be more actionable. It is also referenced in the more general access control issue #1352

@set-reminder 4 weeks @tghosth to revisit if no response

**Comment by tghosth** [comment]
See https://github.com/OWASP/ASVS/issues/1496#issuecomment-1383133452 for historic context on 4.3.3

**Comment by tghosth** [comment]
Closing as I think this is going to get handled here:
https://github.com/OWASP/ASVS/issues/1437#issuecomment-1908030440
